Kaos-Frontend-Coding-Standards.md

```diff
@@ -159,6 +159,24 @@ At Sonatype we value the stability and maintainability of the code base while st
 * For directives, we use an element restriction when "creating a component that is in control of its template" and an attribute restriction when "decorating an existing element with new functionality".
   * This is inline with the Angular documentation as seen in ["When should I use an attribute versus an element?"](https://docs.angularjs.org/guide/directive)
 
+* We place dependent controllers and links below the directive function
+```javascript
+  function Foo(...) {
+    return {
+      controller: FooController,
+      link: FooLink
+    };
+
+    function FooLink(...) {...}
```

Comment:
For consistency sake I think we should nest it within the directive. Also this would help readability for future PRs. For example if you have the link outside of the directive function, and then require use of a directive dependency, nesting the link would cause a large block of deletion and addition. This provides little information as to what was changed within the link.

That being said, this PR does not limit where the link is placed, this is simply an example.